force-app/main/default/classes/TestSampleDataController.cls

```diff
@@ -0,0 +1,17 @@
+@isTest
+private class TestSampleDataController {
+    @isTest
+    static void importSampleData() {
+        Test.startTest();
+        SampleDataController.importSampleData();
+        Test.stopTest();
+
+        Integer propertyNumber = [SELECT COUNT() FROM Property__c];
+        Integer brokerNumber = [SELECT COUNT() FROM Broker__c];
+        Integer contactNumber = [SELECT COUNT() FROM Contact];
+
+        System.assert(propertyNumber > 0, 'Expected cases were created.');
```

Comment:
You probably want to fix the assert messages.

Reply:
done